io.github.chimi70.cosmic-expansion.yml

```diff
@@ -0,0 +1,28 @@
+id: io.github.chimi70.cosmic-expansion
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+base: org.godotengine.godot.BaseApp
+base-version: '3.6'
+sdk: org.freedesktop.Sdk
+command: cosmic-expansion.x86_64
```

Comment:
```suggestion
command: godot-runner.pck
```

Reply:
it godot-runner

https://github.com/flathub/org.godotengine.godot.BaseApp/blob/branch/3.6/org.godotengine.godot.BaseApp.yaml#L6